config/jail.conf

```diff
@@ -731,6 +731,18 @@ logpath  = %(mysql_log)s
 backend  = %(mysql_backend)s
 
 
+# Log wrong MongoDB auth
```

Comment:
imho, this comments belong to the filter, here would be enough a small indication to it... (for details how ... see filter ...)